### ISSUE TITLE
Compile problem

### ISSUE THREAD

**Issue by RGBlu3-pro** [xerootg/btt-s42b-simplefoc#3]
Hi, I can't compile this program. I tried in Platfiormio and Adruino
![image](https://user-images.githubusercontent.com/66820336/212976614-16dee638-ff42-49a8-8b94-1a6fb3eb037c.png)
![image](https://user-images.githubusercontent.com/66820336/212974635-6163af61-56fe-4efb-b3e6-9bb2cd4a909d.png)

**Comment by xerootg** [comment]
It sure looks like the tle5012 driver failed to build. Is this all the logs? I've not built this code since I published it, so I have no idea what it would take to revive it.

**Comment by RGBlu3-pro** [comment]
Yeah, these are all logs.

**Comment by RGBlu3-pro** [comment]
Ok, is done!
Changes:
In TLE5012B.cpp
![image](https://user-images.githubusercontent.com/66820336/213156450-cb18d933-9470-4269-8b8c-5ab0e918bd0d.png)
In TLE5012B.h
![image](https://user-images.githubusercontent.com/66820336/213164348-ca9d17f2-c295-4ca6-9f74-a8ef709eb6e4.png)

**Comment by xerootg** [comment]
Wow, thanks! So it compiles and runs I take it?

**Comment by RGBlu3-pro** [comment]
Not really, probably only in my case, couse I have very terrible boards. One board burned my st link xD
